vfcompositor/Compositor.c

```
@@ -18,130 +21,127 @@
-} WindowStateMapping;
-
-static WindowStateMapping g_window_state_map[MAX_WINDOWS];
-static RustSpinLock* g_text_lock = NULL;
-
-void VFCompositorRequestInit(const char * str) {
-    (void)str;
+// --- Globals are now part of CompositorContext ---
+
+CompositorContext g_compositor_ctx;
+
+void RequestDestroyWindow(CompositorContext* ctx, Window* w) {
```

Clang (14.0.6)

[warning] parameter 'w' is unused

(misc-unused-parameters)